Improve Strimzi network policies for Cluster Operator access (#4005)
strimzi/strimzi-kafka-operator: documentation/modules/ref-operator-cluster.adoc

The timeout for internal operations, in milliseconds. This value should be
increased when using Strimzi on clusters where regular Kubernetes operations take longer than usual (because of slow downloading of Docker images, for example).

`STRIMZI_OPERATOR_NAMESPACE`:: The name of the namespace where the Strimzi Cluster Operator is running.

> This variable should not be configured manually but should use the Kubernetes Downward API.

Reviewer: ```suggestion
Do not configure this variable manually. Use the Kubernetes Downward API.
```

Reviewer: Do we want to show example use of downward api?

Author: > Do we want to show example use of downward api?

No, I do not think we should do it. The user is not expected to touch it. It is in the docs only as an explanation.

+
[source,yaml,options="nowrap"]
----
env:
  - name: STRIMZI_OPERATOR_NAMESPACE
    valueFrom:
      fieldRef:
        fieldPath: metadata.namespace
----

`STRIMZI_OPERATOR_NAMESPACE_LABELS`:: Optional.
The labels of the namespace where the Strimzi Cluster Operator is running.

> These labels will be used to configure the namespace selector in network policies to allow Strimzi Cluster Operator to access the operands only from namespace with these labels.

Reviewer: ```suggestion
Namespace labels are used to configure the namespace selector in network policies to allow the Strimzi Cluster Operator to only have access to the operands from the namespace with these labels.
```

> When not set, the namespace selector in network policies will be configured to allow access from Strimzi Cluster Operator from any namespace in the Kubernetes cluster.

Reviewer: ```suggestion
When not set, the namespace selector in network policies is configured to allow access from the Strimzi Cluster Operator to any namespace in the Kubernetes cluster.
```

Reviewer: Assuming there's a _from_ and _to_ here
